versioned_docs/version-0.2.0/getting-started-with-cloud-clusters/installing-kubeslice/registering-the-worker-cluster.mdx

```diff
@@ -348,7 +348,8 @@ You must apply the values **.yaml** file on each worker cluster to install the S
 
 ## Validating the Slice Operator Installation 
 
-To validate the Slice Operator installation in the registered cluster,check the pods belonging to the namespace `kubeslice-system` by running
+To validate the Slice Operator installation in the registered cluster,check the pods belonging to the 
+namespace `kubeslice-system` by running
```

Comment:
change the  namespace `kubeslice-system` to  the  `kubeslice-system` namespace

Reply:
updated

Reply:
this is not fixed - please fix this comment

Reply:
resolving this as it's outdated but I left a comment on the same change